test-support/helpers/ember-tooltips.js

```diff
@@ -87,3 +101,41 @@ export function assertTooltipVisible(assert, options={}) {
         isTooltipVisible -> ${isTooltipVisible} ;
         isTooltipTetherEnabled -> ${isTooltipTetherEnabled}`);
 }
+
+export function assertTooltipSide(assert, options = {}) {
+  const { side } = options;
+
+  const sideIsValid = (
+    side === 'top' ||
+    side === 'right' ||
+    side === 'bottom' ||
+    side === 'left'
+  );
+
+  /* We make sure the side being tested is valid. We
+  use Ember.assert because assert is passed in from QUnit */
+
+  if (!sideIsValid) {
+    Ember.assert(`You must pass side like assertTooltipSide(assert, { side: 'top' }); Valid options for side are top, right, bottom, and left.`);
+  }
+
+  const expectedSide = options.side || 'top';
```

Comment:
we already have `side` from line 106 I think. `options.side` could be replaced with `side`